SKILL.tsv

```diff
@@ -4983,7 +4983,11 @@ SKILL_20150729_004982	Release Lunacy
 SKILL_20150729_004983	Physical ATK increases, Movement Speed increases, Physical DEF decreases
 SKILL_20150729_004984	Enemy cannot jump when attacked with [Skull Swing].
 SKILL_20150729_004985	$Enchant Fire: Fire Property Resistance Decrease
+<<<<<<< .merge_file_a06728
```

Comment:
Remove "<<<<<<< .merge_file_a06728" along with "=======
+SKILL_20150729_004986	$Decreases the Fire property resistance of nearby enemies when using [Enchant Fire] by 10% for 5 seconds per attribute level.
+>>>>>>> .merge_file_a08556"